### PR TITLE
feat: Implement remaining Notifications UI elements

### PR DESCRIPTION
**Pull request by lynnagara**
No longer using `jsonForm` since we'll have custom UI elements (fine tuning panels)

### CODE REVIEW

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should update CHANGES due to the size of this PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: src/sentry/static/sentry/app/views/settings/account/accountNotifications.jsx [hunk 1]
```diff
@@ -1,21 +1,73 @@
...
-const AccountNotifications = React.createClass({
+import FieldFromConfig from '../components/forms/fieldFromConfig';
+import Panel from '../components/panel';
+import PanelBody from '../components/panelBody';
+import PanelHeader from '../components/panelHeader';
+import PanelFooter from '../components/panelFooter';
+
+import IconChevronRight from '../../../icons/icon-chevron-right';
+
+export default class AccountNotifications extends React.Component {
```
Comment (MaxBittker) [inline src/sentry/static/sentry/app/views/settings/account/accountNotifications.jsx]: 👌

File: src/sentry/static/sentry/app/views/settings/account/accountNotifications.jsx [hunk 2]
```diff
@@ -1,21 +1,73 @@
...
+    field: PropTypes.object.isRequired,
+  };
+
+  constructor(props) {
+    super(props);
+  }
+  render() {
+    let {title, fields, fineTuning} = this.props.field;
+
+    let linkStyle = {
```
Comment (MaxBittker) [inline src/sentry/static/sentry/app/views/settings/account/accountNotifications.jsx]: formatting-wise, what do we want to do with these objects? imo they're kinda noisy to have inline, could we put them at the bottom of the file maybe?

Comment (MaxBittker) [reply src/sentry/static/sentry/app/views/settings/account/accountNotifications.jsx]: also - would we use emotion here?

**Review by MaxBittker — APPROVED** [review]
this looks good to me! + some style questions

File: src/sentry/static/sentry/app/views/settings/account/accountNotifications.jsx (hunk 2 above)
Comment (lynnagara) [reply src/sentry/static/sentry/app/views/settings/account/accountNotifications.jsx]: It won't meet our linting rules if I just move it down (no-use-before-define)

Comment (lynnagara) [reply src/sentry/static/sentry/app/views/settings/account/accountNotifications.jsx]: A styled component might make sense... but I'd prefer not to do that unless it's a more reusable thing

**Pull request merged**